Dockerfile

```diff
@@ -0,0 +1,31 @@
+#  Alfresco Setup build
+
+#FROM 345280441424.dkr.ecr.ap-south-1.amazonaws.com/ark_base:latest
+FROM centos:7
+
+LABEL ORG="Armedia LLC" \
+      APP="Alfresco Setup" \
+      VERSION="1.0" \
+      IMAGE_SOURCE=https://github.com/ArkCase/ark_alfresco_setup \
+      MAINTAINER="Armedia LLC"
+
+WORKDIR /app
+
+# Set default docker_context.
+ARG resource_path=artifacts
+
+COPY ${resource_path}/setup_alfresco.py .
+COPY ${resource_path}/alfresco_setup.properties .
```

Comment:
I thought you didn't need this properties file anymore because you are now using environment variables?

Reply:
yes. removed.